src/sql/parts/connection/connectionDialog/connectionDialogService.ts

```diff
@@ -87,16 +87,31 @@ export class ConnectionDialogService implements IConnectionDialogService {
 		@IClipboardService private _clipboardService: IClipboardService,
 		@ICommandService private _commandService: ICommandService
 	) { }
-
+/**
```

Comment:
Please use // for comments and add some indention just like other comments for methods.

Reply:
/** is fine for large block comments like this

Reply:
`/**` is [TSDoc](https://github.com/Microsoft/tsdoc) instead of a regular comment which like Anthony said is helpful for places like this since it will show up in Intellisense on the method name. Can you add a `@returns` line (look for examples in the TSDoc on other methods) in addition to tabbing this over?

Reply:
Yes. It gives us the intellisense ability over method's implementation. Included @returns as well.